config/webpack/ts-loader-rule.js

```diff
@@ -24,7 +24,7 @@ function getRule(command) {
   // Exclude test specs from type checking during a serve.
   if (command === 'serve') {
     awesomeTypescriptLoaderOptions.reportFiles = [
-      'src/app/**/!(*.spec).ts'
+      'src/app/**/!(*.spec|*.fixture).ts'
```

Comment:
This is the only real feature change (the other changes are to fix `at-loader` reported errors).